csrc/options.h

```diff
@@ -21,6 +21,7 @@ namespace nvfuser {
 //! These can be set through the `NVFUSER_DUMP` environment variable
 //!
 enum class DebugDumpOption {
+  CallStack, //!< Dump the call stack of selected internal function
```

Comment:
It's not about callstack, is it? Otherwise, I'd simply `gdb` and `bt`. I think it's about function tracing similar to `ftrace`, so maybe `DebugDumpOption::FunctionTrace` is a better name.

Reply:
This is definitely a better name, will update that.